packages/zoe/tools/fakeVatAdmin.js

```diff
@@ -54,8 +55,26 @@ function makeFakeVatAdmin(testContextSetter = undefined, makeRemote = x => x) {
         }),
       });
     },
-    createVatByName: _name => {
-      assert.fail(X`createVatByName not supported in fake mode`);
+    createVatByName: name => {
```

Comment:
The code looks redundant. Can we share a single implementation to reduce duplication?

Reply:
The next step is to remove `createVatByName`, so I don't want to put too much energy into it. What about:

```js
createVatByName: name => {
  assert.equal(name, 'zcf', ..);
  return admin.createVat(zcfContractBundle);
}
```

Would that sort of delegate-to-self pattern be readable for a few weeks? Then I'll be able to remove `createVatByName` without also undoing a bunch of de-duplication-driven refactoring.

Reply:
I force-pushed a version with that change

Reply:
That's what I expected.